core/src/main/java/pt/webdetails/cda/connections/dataservices/DataservicesConnectionInfo.java

```diff
@@ -58,10 +48,16 @@ public Properties getProperties() {
 
     DataservicesConnectionInfo that = (DataservicesConnectionInfo) o;
 
-    return getDataServiceName().equals( that.getDataServiceName() );
+    for ( Object key : getProperties().keySet() ) {
+      Object thatName = that.getProperties().get( key );
+      if ( thatName == null || !thatName.equals( getProperties().get( key ) ) ) {
+        return false;
+      }
+    }
+    return true;
```

Comment:
no need to reimplement Properties.equals, could use
`return getProperties().equals( that.getProperties() );`
(properties are initialized in constructor)
this way code coverage would improve